### PR TITLE
Select article comments

### PR DESCRIPTION
**Pull request by Enamulali**
added functionality to view article comments by selected article
including TDD and error 400, and 404(not sure if needed to do these tests again, but did so as new api/path).

### CODE REVIEW

**Review by sirKevlar — COMMENTED** [review]
Looking good so far Enam. I think we need one more 200 test though... It is possible that an article has no comments and therefore would return a status 200, but no comments in the array...

File: __tests__/app.test.js [hunk 1]
```diff
@@ -227,6 +227,83 @@ describe("/api/articles", () => {
...
+          expect(result.body.comments).toEqual(thirdArticle);
+        });
+    });
+    test("status: 200, should return an empty array if no comments exist for chosen article", () => {
+      const articleid = 7;
+      return request(app)
+        .get(`/api/articles/${articleid}/comments`)
+        .expect(200)
+        .then((result) => {
+          expect(result.body.comments).toBeInstanceOf(Array);
```
Comment (emilyb93) [inline __tests__/app.test.js:279]: You've proven its an array but not an empty array

**Review by emilyb93 — APPROVED** [review]
Small change to add to the test file. If this passes, feel free to merge

File: __tests__/app.test.js (hunk 1 above)
Comment (Enamulali) [reply __tests__/app.test.js:279]: Awesome all done

**Pull request merged**